src/recognition/ErrorFunction.java

```diff
@@ -0,0 +1,232 @@
+/**
+ *
+ * @file
+ * @brief Here is Error function for checking accuracy of Classifier work
+ * Also here implemented RandomClassifier which assigns random image to class from
+ * zero  to nine
+ *
+ *
+ * @authors Vladislav Shikhanov
+ *
+ *		  http://move-llc.ru
+ ************************************************************************************/
+package recognition;
+
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Random;
+
+import static recognition.ReadIDX.ImageIDX;
+import static recognition.ReadIDX.testCol;
+
+class ErrorFunction {
+    /*********************** Collections with images definition*********************/
+    private ReadIDX readIDX = new ReadIDX();
+    private ImagesByClass imagesByClass = new ImagesByClass();
+
+    /********************************************************************************
+     *
+     * accordanceAnalyse Method checks all collections with drawings to accordance
+     * assigned images to real values
+     *
+     * @param imagesByClass is compilation of collection of numbers (collection with
+     *                      drawings of 0, of 1 etc)
+     *******************************************************************************/
+    void accordanceAnalyse(ImagesByClass imagesByClass) {
+        for ( int i = 0; i < 10; i++) {
+            Double errors = 0.0;
+            DecimalFormat dec = new DecimalFormat("##0.00");
+            switch (i) {
```

Comment:
The same problem as I mentioned in previous reviews. This switch may grow infinity when you add new numbers. In addition, you perform almost the same code in all branches. Try to generalize it something.
In ideal case, it should look like:
```
errors = errorFunction(some, i);
System.out.println("Errors rate ...");
```
If you don't know how to do it, I can help you.